demosplan/DemosPlanCoreBundle/Controller/User/DemosPlanOrganisationAPIController.php

```diff
@@ -255,8 +255,17 @@ public function wipeOrgaAction(UserHandler $userHandler, string $id): APIRespons
     {
         $orgaId = $id;
         try {
-            $isOrgaDeleted = $userHandler->wipeOrganisationData($orgaId);
-            if ($isOrgaDeleted) {
+            $result = $userHandler->wipeOrganisationData($orgaId);
+
+            if (is_array($result)) {
```

Comment:
I'm not sure, but if it is possible that $result is an empty array then no error message will be added, or not?
Maybe we need a general error message for this case?

Reply:
I think return must be an error array or Orga, Actually it can not be an Empty Array. If array is empty, means orga can be deleted

Reply:
see https://github.com/demos-europe/demosplan-core/pull/2681#discussion_r1477936025